src/coreclr/src/vm/excep.cpp

```diff
@@ -6792,6 +6799,19 @@ AdjustContextForWriteBarrier(
         // put ESP back to what it was before the call.
         SetSP(pContext, PCODE((BYTE*)GetSP(pContext) + sizeof(void*)));
     }
+
+    if ((f_IP >= (void *) JIT_StackProbe) && (f_IP <= (void *) JIT_StackProbe_End)) 
```

Comment:
This is adjusting context for more than just write barrier now. Rename the method for clarity?

Reply:
Makes sense.